assets/dev/js/editor/document/elements/commands/select.js

```diff
@@ -0,0 +1,15 @@
+import CommandBase from 'elementor-api/modules/command-base';
+
+export class Select extends CommandBase {
+	validateArgs( args ) {
+		this.requireContainer( args );
+	}
+
+	apply( args ) {
+		const { containers = [ args.container ], append = false } = args;
+
+		elementor.selection.add( containers, append );
+	}
+}
```

Comment:
If the command called only within another command, and not represent a USER ACTION, then it should be an internal command.

Reply:
@iNewLegend I consulted Mati about that, and we concluded that `select` is a user action and should not be internal.

Reply:
ok, @matipojo just take in account, that `toggle-selection` will be recorded and `select` will recorded... which will be triggered twice.

Reply:
Well, i rechecked the recording mechanism, and we only record the head command, so it's gonna work just fine.